playbooks/group_vars/all

```diff
@@ -1 +1,3 @@
 env: stage
+secure_dir: '../../configuration-secure/ansible'
+local_dir: "{{ secure_dir }}/local"
```

Comment:
@caesar2164 http://docs.ansible.com/ansible/playbooks_best_practices.html#group-and-host-variables